app/models/ability.rb

```diff
@@ -64,7 +67,7 @@ def initialize(user)
            :view_unpublished, :show_eating_habits, :print_applications_eating_habits, :view_hidden], Event
       can :send_email, Email
       can [:manage, :set_contact_person, :set_notes], Request
-      cannot :apply, Event
+      cannot :view_apply_button, Event
```

Comment:
It does, but someone added `:manage` as a permission of Organizers for events. And manage is an alias for all possible actions, so it also allows the organizer `view_apply_button`.
This is a thing that needs refactoring (manage needs to be replaced). I think I already mentioned it in the Deny Apply for Coaches and Organizers PR, but @Lando-L figured out it would be too much things to change, so we decided to make a technical debt issue